config/main.py

```diff
@@ -1500,9 +1500,12 @@ def add_erspan(session_name, src_ip, dst_ip, dscp, ttl, gre_type, queue, policer
         for front_asic_namespaces in namespaces['front_ns']:
             per_npu_configdb[front_asic_namespaces] = ConfigDBConnector(use_unix_socket_path=True, namespace=front_asic_namespaces)
             per_npu_configdb[front_asic_namespaces].connect()
+            # TODO Need to work on API's gather_session_info/validate_mirror_session_config to support multi-asic
```

Comment:
@abdosi Can you check these CLI's for span and erspan w.r.to multi-asic. I have added TODO comments here , I feel we need to have more changes to support multi-asic and handle cases where the src_port list, dst_port etc could be in different front end asics.

Reply:
@abdosi as discussed, I have reverted all changes in mirror config commands from this PR. Could you take a look.
We could take up multi_asic support for these commands in a different PR. Thanks !